src/Common/src/Internal/Runtime/ModuleHeaders.cs

```diff
@@ -58,6 +58,11 @@ public enum ReadyToRunSectionType
         ThreadStaticGCDescRegion = 209,
         ThreadStaticIndex = 210,
 
+        /// <summary>
+        /// Contains pointer relocs to preinitialized data in FrozenObjectRegion
+        /// </summary>
+        GCStaticsPreInitDataRegion = 211,              
```

Comment:
Is this used for anything? It seems like you allocate an ArrayOfEmbeddedObjects for this, but nothing gets actually emitted in the section.